styles.css

```diff
@@ -0,0 +1,19 @@
+img {
+    max-height: 250px;
+
+}
+
+body {
+    background-color: #ADD8E6;
+}
+
+h1, h2 {
+    font-family: 'Poppins', sans-serif;
+    font-family: 'Roboto', sans-serif;
+    font-family: 'Lato', sans-serif;
+
```

Comment:
In CSS, order matters a great deal. Having these three font-families like this just means that the last one will override the two before it. So you will only ever see the Lato font. You can separate them so that you only select the h1 for one font and the h2 for another font or you can select them using a class or an id. We'll get more into that later though.